test/integration/etcd/etcd_storage_path_test.go

```diff
@@ -647,122 +640,31 @@ func TestEtcdStoragePath(t *testing.T) {
 	}
 }
 
-func startRealMasterOrDie(t *testing.T, certDir string) (*allClient, clientv3.KV, meta.RESTMapper) {
-	_, defaultServiceClusterIPRange, err := net.ParseCIDR("10.0.0.0/24")
+func startRealMasterOrDie(t *testing.T) (*allClient, clientv3.KV, meta.RESTMapper, *storagebackend.Config, func()) {
+	restConfig, opts, tearDown := kubeapiservertesting.StartTestServerOrDie(t, nil, framework.SharedEtcd())
```

Comment:
This needs to be config, customizations, start master so you can actually control how master comes up.

Reply:
All possible with flags. Compare media type and authorization.